config/sync/migrate_plus.migration.cs_opportunity_migration.yml

```diff
@@ -0,0 +1,149 @@
+uuid: a8b9c208-5017-4955-8f9a-bb87bf134336
+langcode: en
+status: true
+dependencies:
+  enforced:
+    module:
+      - migrate_plus
+      - migrate_tools
+id: cs_opportunity_migration
+class: null
+field_plugin_method: null
+cck_plugin_method: null
+migration_tags:
+  - JSON
+  - Spotlight
+migration_group: custom_content
+label: 'Import JSON'
+source:
+  plugin: url
+  data_fetcher_plugin: http
+  data_parser_plugin: json
+  urls: 'https://cardinalservice-dev.sites-pro.stanford.edu/opportunity-exporter'
+  item_selector: 0
+  fields:
+    -
+      name: title
+      label: Title
+      selector: title
+    -
+      name: body
+      label: body
+      selector: body
+    -
+      name: su_opp_sponsor
+      label: 'Opportunity Sponsor'
+      selector: su_opp_sponsor
+    -
+      name: su_opp_submissions_open
+      label: 'Opportunity Open Submission'
+      selector: su_opp_submissions_open
+    -
+      name: ctaUrl
+      label: 'Call to Action Link'
+      selector: su_opp_solo_url
+    -
+      name: ctaTitle
+      label: 'Call to Action Title'
+      selector: su_opp_solo_url_1
+    -
+      name: su_opp_contact_email
+      label: 'Contact Email'
+      selector: su_opp_contact_email
+    -
+      name: su_opp_contact_name
+      label: 'Contact Name'
+      selector: su_opp_contact_name
+    -
+      name: su_opp_contact_phone
+      label: 'Contact Phone'
+      selector: su_opp_contact_phone
+    -
+      name: su_opp_eligibility
+      label: 'Eligibility'
+      selector: su_opp_eligibility
+    -
+      name: learnMoreUrl
+      label: 'Learn more'
+      selector: su_opp_learn_more
+    -
+      name: learnMoreTitle
+      label: 'Learn more Title'
+      selector: su_opp_learn_more_1
+    -
+      name: su_opp_location
+      label: 'Location'
+      selector: su_opp_location
+    -
+      name: su_opp_summary
+      label: 'Summary'
+      selector: su_opp_summary
+  ids:
+    title:
+      type: string
+process:
+  title: title
+  body/value: body
+  body/format:
+    plugin: default_value
+    default_value: stanford_html
+  su_opp_sponsor:
+    -
+      plugin: skip_on_empty
+      source: su_opp_sponsor
+      method: process
+    -
+      plugin: entity_generate
```

Comment:
You need to handle multiple values. You will have to delimit the data in the json first. Then this will require something before the `entity_generate` like
```
plugin: explode
delimiter: '|'
```